firmware/src/main.rs

```diff
@@ -59,101 +55,16 @@ fn main() -> ! {
 	static PACKETS: StaticCell<Packets> = StaticCell::new();
 	let packets: &'static Packets = PACKETS.init(Packets::new());
 
-	static QUAT: StaticCell<Unreliable<Quat>> = StaticCell::new();
-	let quat: &'static Unreliable<Quat> = QUAT.init(Unreliable::new());
-
 	static EXECUTOR: StaticCell<Executor> = StaticCell::new();
 	EXECUTOR.init(Executor::new()).run(move |s| {
-		s.spawn(control_task(packets, quat)).unwrap();
 		s.spawn(network_task(packets)).unwrap();
-		s.spawn(imu_task(quat, p.i2c, p.delay)).unwrap();
+		s.spawn(protocol_task(packets)).unwrap();
```

Comment:
use the absolute paths for these tasks, or `use` them just above the EXECUTOR section. IMO that would make it more immediately clear where the tasks are located, without having to hunt through the use statements at the top of the file